### ISSUE TITLE
Project Description 

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#372]
From the client

_When creating a new experiment, the option to add a description is available, however this description isn’t visible anywhere after the cord has been uploaded. Maybe create a tab next to the 2D viewer and Details Viewer called “Description” where users can view and edit the description._

Sounds like they have a point, @syamace-metacell what do you think? cc @afonsobspinto

**Comment by syamace-metacell** [comment]
@neuroBazzi Agreed on this, we don't necessarily have to have it as a tab but we should definitely display it somewhere, along with the other information of the experiment (description, tags, owner). I'm looking at the figma mockups and there is actually a place where all these details go but it wasn't being implemented - not 100% sure why this is since it was before I joined. Do you know? Either way, we could have this or come up with a different solution but definitely show these details yes.

**Comment by syamace-metacell** [comment]
Here's the figma mockups I was talking about
https://github.com/MetaCell/salk-interactive-atlas/assets/139959368/bd7a03c5-2b8c-47a4-92ab-a2c6fc065e24

**Comment by neuroBazzi** [comment]
thanks @syamace-metacell I always missed that, lol, thanks for bringing it up. in principle the modal window with all those info works altho it would be a bit 'hidden' because the user does not know where to click to open in. do you think that adding an 'info' button somewhere that will open that model would make sense? @afonsobspinto I am not sure why we never implemented it, do you recall a reason?

**Comment by syamace-metacell** [comment]
@neuroBazzi Yeah I agree and yes it make sense. we could always have the info button on the top header/nav which opens up the modal

**Comment by afonsobspinto** [comment]
I don't recall any reason, most likely we just missed that from the original mockups. We already have the edit modal in the dashboard so adding it in the experiments page should be straightforward (2h).
Regarding where we should put the modal trigger, I think the info button might be more common/clear to the user.

**Comment by neuroBazzi** [comment]
okthank you
